### PR TITLE
Mobile app navigation

### PR DESCRIPTION
**Pull request by pgiraud**
With this pull request we're changing the way the navigation is managed. The DOM structure is more flexible. Also the animations in the navigation bar are easier to handle.
@elemoine It would be nice if you could give the javascript part a more detailed review.

### CODE REVIEW

**Comment by pgiraud** [comment]
Hum, what's missing is the right nav menu. I'll add it.

File: contribs/gmf/examples/mobileapp.js [hunk 1]
```diff
@@ -64,63 +64,84 @@ app.navDirective = function() {
...
+
+            // deactivate the currently active nav
+            header.find('nav.active').removeClass('active')
+                .addClass('slide-out');
+
+            // show the back button when relevant
+            backButton.toggleClass('active', slid.length > 0);
+
+            // create a new nav
+            nav = $('<nav>');
```
Comment (elemoine) [inline contribs/gmf/examples/mobileapp.js]: `var` is missing here.

**Comment by elemoine** [comment]
The build failed because of a missing `var`.

**Comment by elemoine** [comment]
The logo is still broken for me. The image is there but the browser cannot load it.

File: contribs/gmf/examples/mobileapp.js [hunk 2]
```diff
@@ -64,63 +64,84 @@ app.navDirective = function() {
...
+            nav.append($('<span>', {
+              text: active.attr('data-header-title')
+            }));
+            header.append(nav);
+            window.setTimeout(function() {
```
Comment (elemoine) [inline contribs/gmf/examples/mobileapp.js]: Why do you need to use `setTimeout` here? (Code-smell)

**Comment by elemoine** [comment]
@pgiraud, could you please fix the build, the logo image, squash your commits, and merge? I'll have a closer look at the js code when this is merged.

Thanks for work on this!

**Comment by elemoine** [comment]
I guess you'll also need to rebase this onto master, or the build will probably fail again (because of https://github.com/camptocamp/ngeo/pull/370).

**Comment by elemoine** [comment]
You can also cherry-pick https://github.com/elemoine/ngeo/commit/a216810474bacc8bd5f59910d7e55d5268c113ee before merging. But note that I haven't been able to confirm that my changes pass the build (because of the current build failure).

**Comment by pgiraud** [comment]
The layout doesn't work correctly on small devices. This seem to be due to the fact that the right menu is on top of left menu.

**Comment by elemoine** [comment]
@pgiraud, I wanted to add the mobileapp to the gh-pages branch, as requested by Yves B, but I still have problems with the logo image.

**Comment by pgiraud** [comment]
I tried to modify the image and it still doesn't appear correctly in github. I have no clue about what I'm doing wrong.

**Comment by elemoine** [comment]
Let's merge this.

**Pull request merged**